README.md

```diff
@@ -61,231 +33,158 @@ For more information on the importance of a professional README for your reposit
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+# 📖 Javascript-capstone <a name="about-project"></a>
 
-**[your_project__name]** is a...
+**Javascript-capstone** is built using JavaScript, HTML, and CSS, with webpack as the build tool. It makes use of the TvmazeAPI to retrieve information about each movies, and also utilizes a custom API to store and retrieve comments. 
 
-## 🛠 Built With <a name="built-with"></a>
+**Javascript-capstone** is an Html,CSS&JavaScript-based project
 
+## :hammer_and_wrench: Built With <a name="built-with"></a>
+- HTML
+- CSS
+- JavaScript
 ### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
-  <summary>Client</summary>
+  <summary>Version Control</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://github.com/">Git Hub</a></li>
   </ul>
 </details>
-
 <details>
-  <summary>Server</summary>
+  <summary>Visual Studio Code</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://code.visualstudio.com">Visual Studio Code</a></li>
   </ul>
 </details>
-
 <details>
-<summary>Database</summary>
+  <summary>Client</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://html.com/html5/">HTML5</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **Follow HTML and CSS best practice**
+- **Display Text**
+- **form validation**
+- **Webpack plugins**
+- **Single Page behaviors**
+- **Implementation of ES6 and modules**
+- **Fetch data from API**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [Live Demo]()
```

Comment:
Kindly provide a link/url to the live demo link so that it can lead to the right page when you click on it. This will give your project a more professional outlook and up to standard.

Reply:
Great!